### PR TITLE
docs: fix typo and couple grammatical errors in Readme

### PR DESCRIPTION
**Pull request by hozefaj**
fixed typos and grammatical errors in the docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -265,17 +265,17 @@ $ ./scripts/create-rule.js my-new-rule
...
+These APIs were developed as computer interfaces shifted from buffers (which are text-based and inherently quite accessible) to graphical user interfaces (GUIs). The first attempts to make GUIs accessible involved raster image parsing to recognize characters, words, etc. This information was stored in a parallel buffer and made accessible to assistive technology (AT) devices.
 
 As GUIs became more complex, the raster parsing approach became untenable. Accessibility APIs were developed to replace them. Check out [NSAccessibility (AXAPI)](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/ApplicationKit/Protocols/NSAccessibility_Protocol/index.html) for an example. See [Core Accessibility API Mappings 1.1](https://www.w3.org/TR/core-aam-1.1/) for more details.
 
 ### Browsers
-Browsers support an Accessibility API on a per operating system basis. For instance Firefox implements the MSAA accessibility API on Windows, but does not implement the AXAPI on OSX.
```
Comment (ljharb) [inline README.md]: this comma is correct, since it indicates a pause

**Comment by hozefaj** [comment]
Yeah I guess my editor is adding them in save

**Comment by hozefaj** [comment]
Do you need me to remove it?

**Comment by ljharb** [comment]
Not a huge deal, but nice to avoid unnecessary diffs.

**Comment by hozefaj** [comment]
Will do.

File: README.md [hunk 2]
```diff
@@ -265,17 +265,17 @@ $ ./scripts/create-rule.js my-new-rule
...
+> The accessibility tree and the DOM tree are parallel structures. Roughly speaking the accessibility tree is a subset of the DOM tree. It includes the user interface objects of the user agent and the objects of the document. Accessible objects are created in the accessibility tree for every DOM element that should be exposed to assistive technology, either because it may fire an accessibility event or because it has a property, relationship or feature which needs to be exposed. Generally, if something can be trimmed out it will be, for reasons of performance and simplicity. For example, a <span> with just a style change and no semantics may not get its own accessible object, but the style change will be exposed by other means.
```
[the remark this answers is not in the document]
Comment (hozefaj) [reply README.md]: All changes are done. Ready to be merged.

**Pull request merged**